plugin/build.gradle

```diff
@@ -299,6 +299,19 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.model.MLModelGroupManager',
         'org.opensearch.ml.helper.ModelAccessControlHelper',
         'org.opensearch.ml.action.models.DeleteModelTransportAction.2'
+        'org.opensearch.ml.action.models.DeleteModelTransportAction.2',
+        'org.opensearch.ml.action.tools.ListToolsTransportAction',
```

Comment:
Don't add these class to exclusion list. We should add more test to increase coverage

Reply:
rebase and add unit test